src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -83,6 +83,7 @@
     <Compile Include="$(CommonTestPath)\System\PlatformDetection.cs">
       <Link>Common\System\PlatformDetection.cs</Link>
     </Compile>
+    <Compile Condition="'$(TargetGroup)'=='netstandard1.7'" Include="System\Math.1.7.cs" />
```

Comment:
>Math.1.7.cs [](start = 76, length = 11)

@weshaggard have we decided the convention on how to name this files? I thought that the preferred was to have something like `Math.netstandard.1.7.cs`